qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -1513,23 +1519,23 @@ class Kubernetes implements OrchestratorMain {
 
     List<K8sRole> getClusterRoles() {
         return evaluateWithRetry(2, 3) {
-            def clusterRoles = []
+            List<K8sRole> clusterRoles = []
             client.rbac().clusterRoles().list().items.each {
                 clusterRoles.add(new K8sRole(
                         name: it.metadata.name,
                         namespace: "",
                         clusterRole: true,
                         labels: it.metadata.labels ? it.metadata.labels : [:],
                         annotations: it.metadata.annotations ? it.metadata.annotations : [:],
-                        rules: it.rules.collect {
+                        rules: it?.rules ? it.rules.collect {
```

Comment:
Why do we need that? Collect on null will result in empty slice https://stackoverflow.com/q/30543355 
```suggestion
                        rules: it?.rules.collect {
```

Reply:
I thought so as well, but until I changed it this way it [failed pretty reliably](https://github.com/stackrox/stackrox/pull/5082#discussion_r1129614089).